patchwork/models.py

```diff
@@ -823,6 +823,30 @@ class Meta:
         ]
 
 
+class Note(models.Model):
+    patch = models.ForeignKey(
+        Patch,
+        related_name='note',
+        related_query_name='note',
+        on_delete=models.CASCADE,
+    )
+    submitter = models.ForeignKey(Person, on_delete=models.CASCADE)
+    last_modified = models.DateTimeField(default=tz_utils.now)
```

Comment:
Can we track created date? I'd actually like to add that to all models, so perhaps we could use `created_at` and `updated_at` rather than `last_modified`.

Reply:
To be clear, I'm not suggesting *you* need to add it to all models (though if you wanted to submit another PR doing this I won't complain :innocent:

Reply:
Changing all of the models would be a bit out of scope for this PR but I created a `TimestampMixin` to make it easier to change the other models (perhaps in another PR :eyes: )